samples/bluetooth/peripheral_ots/src/main.c

```diff
@@ -71,19 +80,35 @@ static int ots_obj_created(struct bt_ots *ots, struct bt_conn *conn,
 		return -ENOMEM;
 	}
 
-	if (init->size.alloc > OBJ_MAX_SIZE) {
+	if (add_param->size > OBJ_MAX_SIZE) {
 		printk("Object pool item is too small for Object with %s ID\n",
 		       id_str);
 		return -ENOMEM;
 	}
 
+	if (object_being_created) {
+		created_param->name = object_being_created->name;
+		created_param->size = object_being_created->size;
+		created_param->props = object_being_created->props;
+	} else {
+		index = id - BT_OTS_OBJ_ID_MIN;
+		strcpy(objects[index].name, "");
```

Comment:
```suggestion
		(void)strcpy(objects[index].name, "");
```

Reply:
Sorry, I am not used to this convention, so I keep forgetting :(